build/helpers/contains.js

```diff
@@ -1,6 +0,0 @@
-const util = require('handlebars-utils');
```

Comment:
This file is necessary for handlebars - where did the functionality get moved to?

Reply:
https://github.com/patternfly/patternfly-org/blob/master/packages/patternfly-4/gatsby-theme-patternfly-org/helpers/createHandlebars.js